.github/workflows/_run-pact-provider-tests.yml

```diff
@@ -24,7 +24,8 @@ permissions:
 jobs:
   run-products-as-provider-tests:
     name: Run Products as Provider
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I think an extra line has sneaked in here and in in the other file. In this case it doesn't matter too much (Github Actions yaml is relatively forgiving!) but in other cases it might result in something unexpected! Just something to watch out for in the future.